<ui version="4.0" >
 <class>UnpackDlg</class>
 <widget class="QDialog" name="UnpackDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Unpack an archive</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="4" row="0" column="0" >
    <widget class="QTreeWidget" name="treeFiles" >
     <property name="columnCount" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="pushUnpack" >
     <property name="text" >
      <string>Unpack</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="pushAbort" >
     <property name="text" >
      <string>Abort</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pushHide" >
     <property name="text" >
      <string>Hide</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>202</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
